frontend/lib/src/AppNode.test.ts

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+import { Writer } from "protobufjs"
+import { vectorFromArray } from "apache-arrow"
```

Comment:
Nit/Question: Maybe I'm missing something, but why are these not sorted alphabetically?

Reply:
So that's a tradeoff of how it is configured. I set it up to alphabetize the things that are imported _within_ the import statement, but have it ignore the actual declaration line so that it doesn't fight with the input order by group. The thought being that the group is more important to be able to get an idea of where this import came from, and the alphabetization is just nice to have consistency and a sorting within that import statement. Does that make sense?

Reply:
Oh okay so its not possible to have it sort first by group, then second alphabetically within the group, then alphabetically within the import statements?

Reply:
I have not been able to figure out how to make it do that yet.. :/